volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -447,12 +447,14 @@ def _parse_maple_tree_node(
 
 
 class mm_struct(objects.StructType):
-    def get_mmap_iter(self) -> Iterable[interfaces.objects.ObjectInterface]:
-        """Returns an iterator for the mmap list member of an mm_struct."""
+    def _get_mmap_iter(self) -> Iterable[interfaces.objects.ObjectInterface]:
```

Comment:
This is technically a bigger change in the core API, so would require a version bump (to at least 2.6.0).  However, since this is implemented as a removal of a call, it should be a full version bump so 3.0.0.  To avoid that and only have the smaller version bump, either leave the methods as `get_mmap_iter` or implement a `get_mmap_iter` which simply passes through and adds a deprecation message ([this](https://dev.to/hckjck/python-deprecation-2mof) looks a reasonable guide to follow to do so).  Unfortunately, we have to change the framework version number to track this, and this is why interfaces need careful consideration before being made public...  5;)